pages/common/license.md

```diff
@@ -0,0 +1,19 @@
+# license
+
+> Create license files for open-source projects.
+
+- Create a license:
```

Comment:
That algorithm is lengthy and not really easy to condense. That environmental variable for example I wouldn't mention, that's behavior that's set up once by a person who knows this tool and figures she needs it more frequently. That sounds beyond the tldr experience. The part about the git config could be mentioned because that's easy to get the author extracted.

@waldyrious was that directed at my comment?  I don't see how any following examples contradict? If I get my output to stdout instead of a file I just stream it into a file, no harm done. Of course I'm not really against `and writes to stdout` or something but it's already an often-made assumption.